cmd/ping/kodata/index.html

```diff
@@ -203,20 +203,15 @@ <h2>How does this work?</h2>
     </main>
   </div>
 </body>
-<script src="descs.js"></script>
-<script src="config.js"></script>
 <script type="text/javascript">
 // TODO: Show regions on a map, with lines overlayed according to ping times.
 // TODO: Add an option to contribute times and JS geolocation info to a public BigQuery dataset.
 
 const _MAX_RESULTS = 20; // Only consider most recent results.
+let _ENDPOINTS = {};
```

Comment:
We can perhaps ignore the required JS changes in this PR and focus on the "merging of CLI and the gcping server" part. I could take care of the refactor of frontend in a later PR, once this is merged with the "main" branch.